data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/model/Cluster.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservicemodel.v1.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+@Data
+@Builder
+// required for builder
+@AllArgsConstructor
+// required for mapstruct
+@NoArgsConstructor
+public class Cluster {
+  @NonNull private String accountId;
+  @NonNull private String cluster;
```

Comment:
Can we change the name of this field into "clusterName" ? I was a little bit confused when looking at this change as both the class name and this attribute are called "Cluster". 
Also want to confirm, in the "cluster" field of an environment, we should still just keep the cluster name instead of the whole Cluster object?

Reply:
Well if we still just use the cluster name in the Environment object, I think we can just keep it as "cluster" here to avoid further confusion....or change both into "clusterName" :-D

Reply:
Yeah, it's really inconsistent right now. I'm in favor of being clearer; let's use:

- `clusterName` if it's only the name
- `clusterArn` if it's actually a full ARN
- `cluster` if it's the same format as what the ECS API accepts (i.e. either name of fully qualified ARN)

Reply:
> Also want to confirm, in the "cluster" field of an environment, we should still just keep the cluster name instead of the whole Cluster object?

Environment is a bit of a special case, since it contains `accountId` itself. I think the mapping between types might become really annoying if we make EnvironmentId be something like `(cluster: Cluster, environmentName: String)`